### PR TITLE
🐘 DB Constraints on FlowSession.uuid and FlowRun.status

### PR DESCRIPTION
**Pull request by rowanseymour**
* Make `FlowSession.uuid` non-null, unique and indexed
* Make `FlowRun.status` non-null

I assume this will need to be faked and added manually with CONCURRENTLY

### CODE REVIEW

**Comment by nicpottier** [comment]
Can you make the field non-null concurrently? That's going to be blocking I think.

**Comment by rowanseymour** [comment]
Yeah we have two options there:

1. Just do it. The flow session table is surprising small thanks to aggressive trimming (it's current ~6 million on rapidpro). Maybe we just have to kill mailroom for a few seconds?
2. Add a custom non-null constraint on the table, initially invalid, validated later without locking.

Leaning toward 1.

**Comment by nicpottier** [comment]
I'm ok trying 1, but we'll have to do it during off hours.

File: temba/flows/models.py [hunk 1]
```diff
@@ -2477,10 +2477,6 @@ class FlowSession(models.Model):
     # the flow of the waiting run
     current_flow = models.ForeignKey("flows.Flow", null=True, on_delete=models.PROTECT)
 
-    @classmethod
-    def create(cls, contact, connection):
-        return cls.objects.create(org=contact.org, contact=contact, connection=connection)
```
Comment (rowanseymour) [inline temba/flows/models.py]: removing this as we only create FlowSession instances in tests.. RP no longer writes to sessions (or runs)

**Comment by rowanseymour** [comment]
@nicpottier maybe we can attempt to run this sometime this weekend

**Pull request merged**